components/takgu/game/GameResultList.tsx

```diff
@@ -21,7 +21,11 @@ export default function GameResultList({
     useGameResultList(path);
 
   const page =
-    pathName === '/' ? 'main' : pathName === '/game' ? 'game' : 'profile';
+    pathName === '/takgu'
+      ? 'main'
+      : pathName === '/takgu/game'
+      ? 'game'
+      : 'profile';
```

Comment:
3항연산이 중첩되서 조금 복잡한 것 같아요. 조금 더 단순했으면 하는데 이런 형태는 어떨까요?
```suggestion
  {
    '/takgu': 'main',
    '/takgu/game': 'game'
  }[pathName]||'profile';
```

Reply:
아 확인이 늦어서 죄송합니다 🙇 말씀해주신대로 삼항연산자를 중첩하는 지금의 코드는 코드가 복잡하기도 하고 나중에 사용되는 페이지가 추가되는 경우에 코드 수정이 많이 필요할 것 같아서 객체로 관리하는 것이 좋은 것 같습니다!

다만 역시 말씀해주신대로 조건문 안에 바로 객체를 넣다 보니까 좀 복잡하게 느껴지고 가독성이 좋지 않아지는 것 같아서 아래처럼 따로 객체를 선언하는 방식으로 수정해보았습니다...ㅎㅎ

```typescript
type pageType = 'main' | 'game' | 'profile';

const pageTypeMap: Record<string, pageType> = {
  '/takgu': 'main',
  '/takgu/game': 'game',
};

// 생략

const page: pageType = pageTypeMap[pathName] || 'profile';
```

Reply:
LGTM! 좋아보입니다!